scripts/privyr-generic-cf-integration.js

```diff
@@ -28,6 +28,16 @@ export default class PrivyrGenericCfIntegration {
         return luid;
     }
 
+    _post_xhr_request(post_url, payload, async=false) {
```

Comment:
Will this get compiled to ECMA 5? Because it doesn't support async=Flase in signature

Reply:
Yes.. It will.